scripts/queuestat

```diff
@@ -147,8 +150,7 @@ class Queuestat(object):
         for key, data in cnstat_dict.iteritems():
             if key == 'time':
                 continue
-            index = int(data.queueindex) % (queue_count / 2)
-            table.append((port, data.queuetype + str(index),
+            table.append((port, data.queuetype + str(data.queueindex),
```

Comment:
It looks like we seeing different index number in Counter DB. For example. I dump some sample index value on my test platform. there are 10 UC and 10MC queue. And many queue index are large than 10. So I use int(data.queueindex) % (queue_count / 2)  to find its offset value from 0. How about the index in the COUNTERS_QUEUE_INDEX_MAP in your case?

127.0.0.1:6379[2]> hgetall  COUNTERS_QUEUE_INDEX_MAP
   1) "oid:0x15000000000329"       
   2) "3"
   3) "oid:0x150000000004c3"
   4) "3"
   5) "oid:0x15000000000163"
   6) "0"
   7) "oid:0x150000000001a0"
   8) "19"
   9) "oid:0x1500000000030a"
  10) "12"
  11) "oid:0x1500000000049f"
  12) "8"
  13) "oid:0x150000000001f1"
  14) "18"
  15) "oid:0x150000000004d4"
  16) "19"
  17) "oid:0x15000000000388"

Reply:
thanks for review
here is another PR rised to fix queue index init bug in portsorch:
https://github.com/Azure/sonic-swss/pull/505

Reply:
Look into #505 and will reply soon.

Reply:
Asked a question in #505